.eslintrc.json

```diff
@@ -126,8 +126,9 @@
         "scripts/templates/accessibility_ComponentName_spec.js"
       ],
       "rules": {
+        "flowtype/require-valid-file-annotation": "off",
         "jest/expect-expect": "off",
-        "flowtype/require-valid-file-annotation": "off"
+        "jest/valid-expect": "off"
```

Comment:
Jest assumes that any `expect` is its own global, which isn't true in our Cypress tests (see page_header_spec for the change that necessitated this suppression)

Reply:
should we disable the invalid use case inline vs turning this off globally?

Reply:
Normally I would agree, but this is applying a Jest rule to Cypress tests, where it isn't relevant. Really we could/should turn off all the Jest rules for the Cypress tests, but I was taking a reverse-YAGNI approach to turning those rules off 😅

Reply:
ah i missed that this was scoped to cypress/**/*.js